fdb-relational-grpc/src/main/java/com/apple/foundationdb/relational/jdbc/TypeConversion.java

```diff
@@ -829,4 +827,32 @@ public static Options fromProtobuf(com.apple.foundationdb.relational.jdbc.grpc.v
         }
         return builder.build();
     }
+
+    static int toSqlType(Type type) throws SQLException {
+        switch (type) {
+            case INTEGER:
+                return Types.INTEGER;
+            case LONG:
+                return Types.BIGINT;
+            case STRING:
+                return Types.VARCHAR;
+            case ENUM:
+            case UUID:
+                return Types.OTHER;
+            case BOOLEAN:
+                return Types.BOOLEAN;
+            case ARRAY:
+                return Types.ARRAY;
+            case STRUCT:
+                return Types.STRUCT;
+            case BYTES:
+                return Types.BINARY;
+            case FLOAT:
+                return Types.FLOAT;
+            case DOUBLE:
+                return Types.DOUBLE;
+            default:
+                throw new SQLException("JDBC Type: " + type + " not supported");
+        }
+    }
```

Comment:
The test gap analysis is claiming this method is uncovered. Unless this is a mistake of the code coverage reporting tool, that would seem to suggest we're not really looking at this part of the code base that changed. Do you know what the issue is?